### PR TITLE
Update inline post + Add documentation for HSD cases in tests-dev #2498

### LINKED ISSUE #2477: Update of UPP into UFS weather model
## Description

UPP was recently updated with additional output for SFS. https://github.com/NOAA-EMC/UPP/pull/1066. UFS weather model needs to be updated to easily use these updates in the weather model and g-w. 

## Solution

Update UPP to the latest dev. 

## Related to

. https://github.com/NOAA-EMC/UPP/pull/1066

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
branch.

## Commit Queue Requirements:

- [x] Fill out all sections of this template.
- [x] All sub component pull requests have been reviewed by their code managers.
- [x] Run the full Intel+GNU RT suite (compared to current baselines) on either Hera/Derecho/Hercules
- [x] Commit 'test_changes.list' from previous step
---
## Description:
This PR updates inline post  for SFS and GEFS developments by:

- Updating the upp submodule within FV3 with recent changes from the UPP repository 

- Updating inline post read interface

### Commit Message:

```
* UFSWM - Documentation for HSD cases 
  * FV3 - 
    * upp - update upp hash to ce258fca with update inline post interface.
```

### Priority:

* High: Support both GEFS and SFS developments

### UFSWM:

* Closes #2477

### Sub component Pull Requests:

* FV3: NOAA-EMC/fv3atm#880

### UFSWM Blocking Dependencies:

* None

---
### Regression Test Changes (Please commit test_changes.list):

* PR Updates/Changes Baselines.

### Input data Changes:

* None.

### Library Changes/Upgrades:

* No Updates
  
---

## Testing Log:
- RDHPCS
  - [x] Hera
  - [x] Orion
  - [x] Hercules
  - [x] Jet
  - [x] Gaea
  - [ ] Derecho
- WCOSS2
  - [x] Dogwood/Cactus
  - [ ] Acorn
- [ ] CI
- [ ] opnReqTest (complete task if unnecessary)

### CODE REVIEW

File: tests/tests/rap_clm_lake_debug [hunk 1]
```diff
@@ -23,3 +23,4 @@ export CCPP_SUITE=FV3_RAP_clm_lake
 export LKM=1
 export IOPT_LAKE=2
 export FRAC_ICE=.true.
+export WRITE_DOPOST=.false.
```
Comment (WenMeng-NOAA) [inline tests/tests/rap_clm_lake_debug:26]: @jkbk2004 @ulmononian Can you also incorporate this change in PR #2461? With the recent UPP updates, the inline post is not working for this test anymore.

**Comment by jkbk2004** [comment]
@WenMeng-NOAA can you sync up branches? We like to commit this pr today.

**Comment by BrianCurtis-NOAA** [comment]
@jkbk2004 How is this PR ready for commit?

Before this PR can make the top of the commit queue it needs:
* upstream review approvals
* run on a platform with GNU support for testing
* updated test_changes.list after testing

These are all in checkboxes at the top in the template.

Please make sure the template is fully checked off and double checked by EPIC CM's before adding it to the top of the commit queue. As soon as a PR makes the top of the commit queue and it's synced I should be able to kick off official testing on my platforms.

**Comment by jkbk2004** [comment]
> @jkbk2004 How is this PR ready for commit?
> 
> Before this PR can make the top of the commit queue it needs:
> 
> * upstream review approvals
> * run on a platform with GNU support for testing
> * updated test_changes.list after testing
> 
> These are all in checkboxes at the top in the template.
> 
> Please make sure the template is fully checked off and double checked by EPIC CM's before adding it to the top of the commit queue. As soon as a PR makes the top of the commit queue and it's synced I should be able to kick off official testing on my platforms.

@BrianCurtis-NOAA We will make sure about the requirement.

**Comment by jkbk2004** [comment]
@FernandoAndrade-NOAA @BrianCurtis-NOAA rap, hrrr, regional cases are changing. new bl_date is needed.

**Comment by jkbk2004** [comment]
@gspetro-NOAA we expect to combine in #2498 (doc update) to this pr. As tests go, we can continue to try to combine whenever the pr is ready.

**Comment by jkbk2004** [comment]
@gspetro-NOAA #2498 is merged into this pr.

**Comment by FernandoAndrade-NOAA** [comment]
@BrianCurtis-NOAA just making sure, are we skipping Acorn this PR?

**Comment by FernandoAndrade-NOAA** [comment]
From the daily tag up, we will be skipping Acorn this PR.

**Comment by BrianCurtis-NOAA** [comment]
Yes skipping acorn.

**Review by jkbk2004 — APPROVED** [review]

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Pull request merged**